config.js

```diff
@@ -1,7 +1,7 @@
 var config = {};
 // Accounts to read the last tweet from. The first one in the list will be
 // spoken by the party parrot.
-config.twitter = process.env.TTC_BOTS || 'tinycarebot,selfcare_bot,magicrealismbot';
+config.twitter = process.env.TTC_BOTS || 'tinycarebot,selfcare_bot,magicrealismbot,5minselfcare';
```

Comment:
I would prefer the `5minselfcare` over ...lets say.... the `magicrealismbot`
@trymnilsen Maybe you want to change your PR?